assets/fonts/almarai/style.css

```diff
@@ -0,0 +1,32 @@
+/* #### Generated By: http://www.cufonfonts.com #### */
+
+    @font-face {
+    font-family: 'Almarai Regular';
+    font-style: normal;
+    font-weight: normal;
+    src: local('Almarai Regular'), url('Almarai-Regular.woff') format('woff');
+    }
+    
+
+    @font-face {
+    font-family: 'Almarai Light';
+    font-style: normal;
+    font-weight: normal;
+    src: local('Almarai Light'), url('Almarai-Light.woff') format('woff');
+    }
+    
+
+    @font-face {
+    font-family: 'Almarai Bold';
+    font-style: normal;
+    font-weight: normal;
+    src: local('Almarai Bold'), url('Almarai-Bold.woff') format('woff');
+    }
+    
+
+    @font-face {
+    font-family: 'Almarai ExtraBold';
+    font-style: normal;
+    font-weight: normal;
+    src: local('Almarai ExtraBold'), url('Almarai-ExtraBold.woff') format('woff');
+    }
```

Comment:
Each font face should declare their correct weight, so that each is used when needed. Otherwise it will load only one and the browser will do their best to make them lighter or bolder.

E.g.:
```css
@font-face {
    font-family: 'Almarai Bold';
    font-style: normal;
    font-weight: bold;
    src: local('Almarai Bold'), url('Almarai-Bold.woff') format('woff');
}
```